src/VisualStudio/IntegrationTest/TestUtilities/InProcess/InProcComponent.cs

```diff
@@ -124,10 +124,29 @@ protected static void WaitForApplicationIdle(TimeSpan timeout)
 #pragma warning restore VSTHRD001 // Avoid legacy thread switching APIs
 
         protected static void WaitForSystemIdle()
+        {
 #pragma warning disable VSTHRD001 // Avoid legacy thread switching APIs
-            => CurrentApplicationDispatcher.Invoke(() => { }, DispatcherPriority.SystemIdle);
+            CurrentApplicationDispatcher.Invoke(() => { }, DispatcherPriority.SystemIdle);
 #pragma warning restore VSTHRD001 // Avoid legacy thread switching APIs
 
+            // Work around GitHub Copilot creating output windows and stealing focus
+            JoinableTaskFactory.Run(async () =>
```

Comment:
📝 For the old integration test framework, I would relocate this code here:

https://github.com/dotnet/roslyn/blob/cb3137e1e5a79c34c5ad99d9532f164893b0a3f8/src/VisualStudio/IntegrationTest/TestSetup/IntegrationTestServicePackage.cs#L19-L32

Reply:
Ah excellent; I was trying to find something similar but couldn't.

Reply:
Going a different direction, so resolving this for now.

Reply:
This in the end wasn't useful -- it appeared that this package load is happening under a synchronous block due to the command fetch, so in the end so "let the product just run until the output window is opened" isn't something that can be sanely done. So leaving it this way for now, not that I wouldn't have preferred this.